src/Entity/Advisor.php

```diff
@@ -69,6 +69,17 @@ class Advisor
      */
     private $boards;
 
+    /**
+     * @ORM\Column(type="text", nullable=true)
+     */
+    private $commentary;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="App\Entity\Resume", inversedBy="advisor")
```

Comment:
Here I see an issue with attribute naming, I believe there is a collection of advisors in the entity resume, that will be retrieved by the attribute advisor**s** . Please fix this little issue

Reply:
I made mistakes when creating the relation between Advisor and Resume. It should have been a OnetoMany from Advisor